### PR TITLE
1140 app metrics: add timeexecution and health endpoints

### PR DESCRIPTION
**Pull request by turtle321**

### CODE REVIEW

**Comment by turtle321** [comment]
Related PR in Puppet:
https://gitlab.cern.ch/ai/it-puppet-hostgroup-inspire/merge_requests/80

File: inspirehep/config.py [hunk 1]
```diff
@@ -1623,3 +1623,10 @@
                'arxiv_eprints', 'public_notes', 'number_of_pages',
                'publication_info', 'earliest_date']
 }
+
+
+# App metrics
+# ===========
+APPMETRICS_ENABLED = False
```
Comment (michamos) [inline inspirehep/config.py]: is this a feature flag? then it should follow the pattern and be something like `FEATURE_FLAG_ENABLE_APPMETRICS` and be defined close to the others.

File: inspirehep/modules/theme/views.py [hunk 2]
```diff
@@ -344,6 +345,24 @@ def ping():
     return 'OK'
 
 
+@blueprint.route('/health')
+@time_execution
+def health():
```
Comment (michamos) [inline inspirehep/modules/theme/views.py]: not sure this is the right place for it, as `theme` is the old ui that is being replaced by the new react one. @drjova ?

Comment (ammirate) [reply inspirehep/modules/theme/views.py]: Same for the above `ping`, which is used from nagios to check nginx status.
Shell we move them both somewhere else?

File: inspirehep/config.py (hunk 1 above)
Comment (turtle321) [reply inspirehep/config.py]: True that. Done.

File: inspirehep/modules/theme/views.py (hunk 2 above)
Comment (turtle321) [reply inspirehep/modules/theme/views.py]: And this is also the place where we serve the root '/'. So we will move them together, but not in this PR.

**Comment by ammirate** [comment]
ci merge please

**Comment by turtle321** [comment]
ci merge please

**Pull request merged**